packages/expo-router/src/toolbar/elements.tsx

```diff
@@ -13,41 +12,227 @@ import {
   type LinkMenuProps,
 } from '../link/elements';
 
-export type ToolbarMenuProps = LinkMenuProps;
+/**
+ * For available props, see [LinkMenuProps](./router/#linkmenuprops).
+ *
+ * @platform ios
+ */
+export interface ToolbarMenuProps extends LinkMenuProps {
+  /**
+   * Whether the button shares the background with adjacent toolbar items.
+   *
+   * **important**: Text buttons cannot share the background.
+   *
+   * Only available for root level menus.
+   *
+   * @see https://developer.apple.com/documentation/uikit/uibarbuttonitem/sharesbackground
+   *
+   * @default true
+   *
+   * @platform iOS 26+
+   */
+  sharesBackground?: boolean;
+  /**
+   * Whether to hide the shared background when `sharesBackground` is enabled.
+   *
+   * Only available for root level menus.
+   *
+   * @see https://developer.apple.com/documentation/uikit/uibarbuttonitem/hidessharedbackground
+   *
+   * @platform iOS 26+
+   */
+  hidesSharedBackground?: boolean;
+}
+
+/**
+ * Adds a context menu for to a toolbar.
+ *
+ * @example
+ * ```tsx
+ * <Toolbar>
+ *   <Toolbar.Menu title="Options">
+ *     <Toolbar.MenuAction title="Action 1" onPress={() => {}} />
+ *     <Toolbar.MenuAction title="Action 2" onPress={() => {}} />
+ *   </Toolbar.Menu>
+ * </Toolbar>
+ * ```
+ *
+ * For available props, see [LinkMenuProps](./router/#linkmenuprops).
```

Comment:
```suggestion
```

The link is duplicated. Let's remove it from here.

<img width="1288" height="440" alt="CleanShot 2025-12-16 at 16 39 11" src="https://github.com/user-attachments/assets/2e066d24-bc0e-46ea-9121-f3eaaaf169e7" />